Makefile

```diff
@@ -41,7 +41,7 @@ UNIT_TESTSRC := $(call rwildcard, $(UNIT_TESTSDIR), *.c)
 # Specify the r.e. so it will only pick-up sources that are common to multiple
 # unit-tests (and won't pick-up test-specific common files, e.g.
 # btree_test_common.c)
-COMMON_UNIT_TESTSRC := $(shell find $(UNIT_TESTSDIR) -name "*tests_common.c")
+COMMON_UNIT_TESTSRC := $(wildcard $(UNIT_TESTSDIR)/*tests_common.c)
```

Comment:
Comes about due to changes introduced under earlier PR integrated at SHA 615050a to remove use of `shell` built-ins in favor of using `Makefile` built-in commands.